src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -198,7 +198,33 @@ def default_api_version(self):
         }),
         ResourceType.DATA_KEYVAULT: '2016-10-01',
         ResourceType.DATA_STORAGE: '2017-11-09',
-        ResourceType.DATA_COSMOS_TABLE: '2017-04-17'
+        ResourceType.DATA_COSMOS_TABLE: '2017-04-17',
+        ResourceType.MGMT_MONITOR: SDKProfile('2019-06-01', {
```

Comment:
With Monitor Support in 2019-03-01-hybrid, 2018-03-01-hybrid and 2017-03-09-profile, how can you test these commands in different profiles?

Reply:
removed